pkg/collector/daemonset.go

```diff
@@ -46,7 +46,7 @@ func DaemonSet(cfg config.Config, logger logr.Logger, otelcol v1alpha1.OpenTelem
 			Template: corev1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels:      labels,
-					Annotations: otelcol.Annotations,
```

Comment:
Could you also add an entry to the changelog (perhaps after this is merged?) about this? This can be considered a breaking change.

Reply:
You mean like a separate PR after this? I can do that. I could also add it to changelog right now under `(unreleased)` version if that's ok?

Reply:
That would be fine as well.

Reply:
Added